doc/host-setup.md

```diff
@@ -0,0 +1,64 @@
+# Preparing the host for DRBD
+
+Piraeus depends on DRBD for data replication across multiple nodes. [DRBD] is a kernel module that must be loaded on
+every node that should access the storage. Depending on your exact host OS and set up, you can choose from one of the
+following options to load DRBD:
+
+[DRBD]: https://github.com/LINBIT/drbd/
+
+## Build and load DRBD using the Kernel Module Injection Image (Preferred)
+
+Every satellite container created by the operator will use an [InitContainer] to ensure DRBD is available. This init
+step can also compile and load DRBD directly. To build DRBD, the container needs access to the kernel sources. It will
+look for the sources on the host under `/usr/src` and `/lib/modules/$(uname -r)`. Most distributions provide a convenient package
+for the kernel source:
+
+* Ubuntu: `apt-get install linux-headers-$(uname -r)`
+* CentOS: `yum install kernel-devel-$(uname -r)`
+
+Install these packages and use the injector image matching your distribution:
+
+* CentOS 7: `quay.io/piraeusdatastore/drbd9-centos7`
+* CentOS 8: `quay.io/piraeusdatastore/drbd9-centos8` (**Not** guaranteed to work on CentOS 8 Stream)
```

Comment:
A user reported that it works with the following Kernel, more info at: https://github.com/piraeusdatastore/piraeus-operator/issues/137

```
$ uname -r                                                         
4.18.0-240.1.1.el8_3.x86_64
```

Reply:
Or: 

See https://github.com/piraeusdatastore/piraeus-operator/issues/137 for more info.